src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -212,6 +212,8 @@ public void Stop()
                 {
                     instance.Dispose();
                 }
+
+                IsRunning = false;
```

Comment:
Yes, I discovered this in the test I wrote where I call Stop and I expect IsRunning to be false at that point (test needs to wait for the stop to finish). Good to fix, but doesn't really affect users currently.